lib/index.js

```diff
@@ -3,25 +3,11 @@
  * https://github.com/joyent/node-http-signature
  * Copyright 2012 Joyent, Inc.  All rights reserved.
  *
- * Copyright (c) 2018-2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2018-2022 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
 
-const env = require('./env');
-let assert;
-let Url;
-if(env.nodejs) {
-  assert = require('assert-plus');
-  Url = require('url').URL;
-} else {
-  Url = URL;
-  // assertions disabled in browsers
-  assert = new Proxy({}, {
-    get() {
-      return () => {};
-    }
-  });
-}
+const {assert, Url} = require('./util.js');
```

Comment:
Note that `URL` is a global on node 12 and browsers ... so this isn't even necessary.

```suggestion
const Url = URL;
const {assert} = require('./util.js');
```